src/main/drivers/dshot_command.c

```diff
@@ -209,6 +211,11 @@ void dshotCommandWrite(uint8_t index, uint8_t motorCount, uint8_t command, dshot
     }
 
     if (commandType == DSHOT_CMD_TYPE_BLOCKING) {
+        // Fake command in queue. Blocking commands are launched from cli, and no inline commands are running
+        for (uint8_t i = 0; i < motorDeviceCount(); i++) {
+            commandQueue[commandQueueTail].command[i] = ((i == index) || (index == ALL_MOTORS)) ? command : DSHOT_CMD_MOTOR_STOP;
```

Comment:
```suggestion
            commandQueue[commandQueueTail].command[i] = (i == index || index == ALL_MOTORS) ? command : DSHOT_CMD_MOTOR_STOP;
```

Reply:
Sorry missed this one in previous review.

Reply:
fixed